source/scripts/app.js

```diff
@@ -85,16 +86,31 @@ function createCookbook() {
 function bindCreateCookbookSave() {
   "use strict";
   let shadow = document.querySelector("create-cookbook").shadowRoot;
-  let buttonHandler = shadow.getElementById("save-button");
-  buttonHandler.addEventListener("click", async () => {
+  let saveButton = shadow.getElementById("save-button");
+  let cancelButton = shadow.getElementById("cancel-button-container");
+
+  saveButton.addEventListener("click", async () => {
     let title = shadow.getElementById("title-input").value;
 
     if (title !== "") {
       let description = shadow.getElementById("description-input").value;
       await indexedDb.createCookbook(title, description);
+      populateCookbooksPage();
```

Comment:
We have to wait for the cookbooks to populate before navigating to the My Cookbooks page because if the HTML elements are loading while the user is on the My Cookbooks page, it'll cause screen flicker
```suggestion
      await populateCookbooksPage();
```